src/utils/untar.js

```diff
@@ -0,0 +1,316 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It would be good to have at least one simple test to exercize this code. I'm fine with a small unit test and a small tar file checked in in the repository.

Reply:
The fetchSource test which I added exercises it. But I can add another test which runs just the untar code.